ecommerce-ios/Sources/Search/Row/SearchItemRow.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  SearchItemRow.swift
+//  ecommerce-ios
+//
+//  Created by Nguyen M. Tam on 13/05/2021.
+//
+
+import SwiftUI
+
+struct SearchItemRow: View {
```

Comment:
Please take a look at this, it should be the way we make collectionView https://www.hackingwithswift.com/quick-start/swiftui/how-to-position-views-in-a-grid-using-lazyvgrid-and-lazyhgrid